src/gui_common/TweakedColourPicker.cs

```diff
@@ -0,0 +1,310 @@
+﻿using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   Tweaked color picker defines some custom ColorPicker functions.
+/// </summary>
+public class TweakedColourPicker : ColorPicker
+{
+    private static readonly List<KeyValuePair<string, Color[]>> PresetsStorage
```

Comment:
An explanation for what the dictionary key is/does in the summary could go a long way.